Translations/pt_BR.po

```diff
@@ -4769,31 +4773,31 @@ msgstr ""
 
 #: Source/monstdat.cpp:154
 msgid "Reaper"
-msgstr ""
+msgstr "Ceifador"
 
 #: Source/monstdat.cpp:469
 msgid "Gharbad the Weak"
 msgstr "Gharbad, o Fraco"
 
 #: Source/monstdat.cpp:471 Source/quests.cpp:42
 msgid "Zhar the Mad"
-msgstr "\"Zhar, o Louco"
+msgstr "Zhar, o Louco"
 
 #: Source/monstdat.cpp:472
 msgid "Snotspill"
-msgstr ""
+msgstr "Kaikatarro"
 
 #: Source/monstdat.cpp:473
 msgid "Arch-Bishop Lazarus"
 msgstr "Arcebispo Lázarus"
 
 #: Source/monstdat.cpp:474
 msgid "Red Vex"
-msgstr ""
+msgstr "Vex Vermelho"
```

Comment:
The literal meaning of "Vex" makes some sense for her name I think? Also, should be feminine:
```suggestion
msgstr "Aflição Vermelha"
```